tests/callbacks/test_early_stopping.py

```diff
@@ -183,7 +181,7 @@ def training_epoch_end(self, outputs):
 
 
 def test_pickling(tmpdir):
-    early_stopping = EarlyStopping()
+    early_stopping = EarlyStopping("val_loss")
```

Comment:
Can we use "foo" instead of "val_loss" in all the instances where the value does not matter?